src/ontology/mondo-edit.obo

```diff
@@ -339168,7 +339166,7 @@ xref: MEDGEN:900671 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MESH:C565736 {source="MONDO:equivalentTo"}
 xref: OMIM:604916 {source="MONDO:equivalentObsolete", source="Orphanet:2729/e", source="Orphanet:2729"}
 xref: OMIM:616580 {source="MONDO:equivalentTo", source="Orphanet:453504", source="Orphanet:453504/e"}
-xref: Orphanet:2729 {source="MONDO:equivalentObsolete", source="MONDO:preferredExternal", source="OMIM:604916"}
+xref: Orphanet:2729 {source="MONDO:equivalentObsolete", source="OMIM:604916"}
```

Comment:
This term is not obsolete in orphanet: https://www.orpha.net/en/disease/detail/2729?name=2729&mode=orpha

Reply:
I did not see the message that "This entity has been excluded from the Orphanet nomenclature of rare diseases and moved to [Neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-skeletal anomalies syndrome](https://www.orpha.net/en/disease/detail/453499)"
My apologies. What you did, using "MONDO:equivalentObsolete" was correct. Thank you!

Reply:
Ok, no further changes needed here.